packages/core/src/amazonqFeatureDev/session/sessionState.ts

```diff
@@ -288,6 +300,15 @@ abstract class CodeGenBase {
                 }
                 case CodeGenerationStatus.PREDICT_READY:
                 case CodeGenerationStatus.IN_PROGRESS: {
+                    if (codegenResult.codeGenerationStatusDetail) {
```

Comment:
Should we add some tests?

Reply:
Tests are being added per standup share today.